lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -245,6 +251,12 @@ class AzurePlatformSchema:
             validate=validate.Regexp(constants.GUID_REGEXP),
         ),
     )
+    cloud: str = field(
```

Comment:
1. Try if it's possible to inherit the CloudEndpoints, and add schema annotations on it, like,

```python
@dataclass_json()
@dataclass
class CloudEntry(CloudEndpoints):
    ...
```

2. Make it type like `Union[str, CloudEntry]`. If it's a string, parse it to CloudEntry. For other places, just pass the CloudEntry object in, so it can retrieve any url in it.

Reply:
I didn't change the location part yet, could you please check current change for this part? thanks!